### PR TITLE
Add release notes for JS provider 1.0.0-preview.3

### PR DESCRIPTION
**Pull request by Eskibear**
Main enhancements
- dynamic refresh
- configuration object

### CODE REVIEW

File: releaseNotes/JavaScriptProvider.md [hunk 1]
```diff
@@ -2,6 +2,12 @@
...
+- Added support for consuming settings as an object. A new API, `constructConfigurationObject`, has been added to construct an object based on the hierarchical keys of the loaded settings. It helps to minimize code changes when switching from local configuration to Azure App Configuration. [#49](https://github.com/Azure/AppConfiguration-JavaScriptProvider/pull/49)
+- The `load` method is usually called in the application's startup code path. Unhandled exceptions in the `load` method can cause the application to crash, resulting in crash loops as orchestrators attempt to restart the application. To mitigate back-to-back crash loops and prevent overloading the server with requests, a minimum time is waited to propagate fatal errors. [#35](https://github.com/Azure/AppConfiguration-JavaScriptProvider/pull/35)
```
Comment (zhiyuanliang-ms) [inline releaseNotes/JavaScriptProvider.md:9]:
```suggestion
- Added error handling delay for `load`. The `load` method is usually called in the application's startup code path. Unhandled exceptions in the `load` method can cause the application to crash, resulting in crash loops as orchestrators attempt to restart the application. To mitigate back-to-back crash loops and prevent overloading the server with requests, a minimum time is waited to propagate fatal errors. [#35](https://github.com/Azure/AppConfiguration-JavaScriptProvider/pull/35)

```

File: releaseNotes/JavaScriptProvider.md [hunk 2]
```diff
@@ -2,6 +2,13 @@
...
+- Added support for consuming settings as an object. A new API, `constructConfigurationObject`, has been added to construct an object based on the hierarchical keys of the loaded settings. It helps to minimize code changes when switching from local configuration to Azure App Configuration. [#49](https://github.com/Azure/AppConfiguration-JavaScriptProvider/pull/49)
+- Added error handling delay for `load` method. The `load` method is usually called in the application's startup code path. Unhandled exceptions in the `load` method can cause the application to crash, resulting in crash loops as orchestrators attempt to restart the application. To mitigate back-to-back crash loops and prevent overloading the server with requests, a minimum time is waited to propagate fatal errors. [#35](https://github.com/Azure/AppConfiguration-JavaScriptProvider/pull/35)
```
Comment (zhenlan) [inline releaseNotes/JavaScriptProvider.md:9]: Not really a customer facing feature. I would skip it from the release notes.

File: releaseNotes/JavaScriptProvider.md [hunk 3]
```diff
@@ -2,6 +2,13 @@
...
 [Source code][source_code] | [Package (npm)][package] | [Samples][samples]
 
+## 1.0.0-preview.3 - March 21, 2023
+## Enhancements
+- Added support for updating configuration on demand without causing an application to restart. Dynamic refresh can be enabled by passing `RefreshOptions` when loading the configuration. A new API, `refresh`, has been added to trigger the refresh operation. When this API is called, it checks the server for any new changes to the `watchedSettings` after a specified time interval, `refreshIntervalInMs`. Once there is a change detected on a watched setting, all loaded settings will be updated. See usage in [this example](https://github.com/Azure/AppConfiguration-JavaScriptProvider/blob/main/examples/refresh.mjs). [#21](https://github.com/Azure/AppConfiguration-JavaScriptProvider/pull/21)
+- Added support for consuming settings as an object. A new API, `constructConfigurationObject`, has been added to construct an object based on the hierarchical keys of the loaded settings. It helps to minimize code changes when switching from local configuration to Azure App Configuration. [#49](https://github.com/Azure/AppConfiguration-JavaScriptProvider/pull/49)
```
Comment (zhenlan) [inline releaseNotes/JavaScriptProvider.md:8]: Added support for consuming ~settings~ _configuration_ as an object. A new API, `constructConfigurationObject`, has been added to construct a _configuration_ object based on the ~hierarchical keys of the~ key-values loaded ~settings~ _from Azure App Configuration_. It ~helps to~ minimizes _necessary_ code changes _for applications that were using_ ~when switching from local~ _JSON_ configuration _file_ to _adopt_ Azure App Configuration.

Comment (Eskibear) [reply releaseNotes/JavaScriptProvider.md]: updated in c880a61

File: releaseNotes/JavaScriptProvider.md (hunk 2 above)
Comment (Eskibear) [reply releaseNotes/JavaScriptProvider.md]: removed in f4af1bf

File: releaseNotes/JavaScriptProvider.md [hunk 4]
```diff
@@ -2,6 +2,13 @@
 
 [Source code][source_code] | [Package (npm)][package] | [Samples][samples]
 
+## 1.0.0-preview.3 - March 21, 2023
+## Enhancements
+- Added support for updating configuration on demand without causing an application to restart. Dynamic refresh can be enabled by passing `RefreshOptions` when loading the configuration. A new API, `refresh`, has been added to trigger the refresh operation. When this API is called, it checks the server for any new changes to the `watchedSettings` after a specified time interval, `refreshIntervalInMs`. Once there is a change detected on a watched setting, all loaded settings will be updated. See usage in [this example](https://github.com/Azure/AppConfiguration-JavaScriptProvider/blob/main/examples/refresh.mjs). [#21](https://github.com/Azure/AppConfiguration-JavaScriptProvider/pull/21)
```
[the remark this answers is not in the document]
Comment (Eskibear) [reply releaseNotes/JavaScriptProvider.md]: simplified in ff6e689

File: releaseNotes/JavaScriptProvider.md [hunk 5]
```diff
@@ -2,6 +2,11 @@
 
 [Source code][source_code] | [Package (npm)][package] | [Samples][samples]
 
+## 1.0.0-preview.3 - March 21, 2023
+## Enhancements
```
Comment (zhenlan) [inline releaseNotes/JavaScriptProvider.md:6]: Should this be 3 #?

**Review by zhenlan — APPROVED** [review]

File: releaseNotes/JavaScriptProvider.md (hunk 5 above)
Comment (Eskibear) [reply releaseNotes/JavaScriptProvider.md:6]: Good catch, that was a miss.

Comment (Eskibear) [reply releaseNotes/JavaScriptProvider.md]: fixed in 3cf6116

**Pull request merged**